conda/recipes/libcuvs/meta.yaml

```diff
@@ -63,7 +63,8 @@ outputs:
         - ninja
         - {{ stdlib("c") }}
       host:
-        - libraft ={{ minor_version }}
+        - librmm ={{ minor_version }}
+        - libraft-headers-only ={{ minor_version }}
```

Comment:
We can migrate `libraft` to `libraft-headers-only` because there is no longer a requirement for `libraft` shared objects. This reduces the total dependency tree size for `libcuvs` because users do not need to download `libraft` binaries.

Reply:
I understand the change to remove libraft binaries, but I’d prefer to use libraft-headers instead of libraft-headers-only. Users of raft should be able to get the math libs transitively so they aren’t stuck having to redefine those all in the recipe.

Reply:
cuVS links directly to the CUDA math libraries so we should not remove them here: https://github.com/rapidsai/cuvs/blob/c40abaeb0e9bfa3f7e13db35cc30105574412952/cpp/CMakeLists.txt#L453-L459

I can switch to `libraft-headers` if we are still using math libraries _through_ RAFT (I don't know whether that is true) but I don't know if the header-only pieces of libraft are using math libraries.

Reply:
Usage of all math libs in cuvs are dome through RAFT APIs. I suspect there might be 1 or 2 places in legacy code that might still be calling math libs directly, but those ultimately will be updated to use raft. RAFT intentionally provides abstractions around the math libs so that users have a single cohesive API available for writing algorithms. Does that make sense?

Reply:
```suggestion
        - libraft-headers ={{ minor_version }}
```

Reply:
Yes, I think that makes sense. Changes applied.

Reply:
Thanks!